pkg/crd/manifest.go

```diff
@@ -14,28 +14,63 @@
  * limitations under the License.
  */
 
-package core
+package crd
 
 import (
 	"errors"
 	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/jinzhu/copier"
+	"github.com/projectriff/riff/pkg/fileutils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"net/url"
 	"path/filepath"
+	"strings"
+)
 
-	"github.com/projectriff/riff/pkg/fileutils"
+type ResourceChecks struct {
+	Kind     string               `json:"kind,omitempty"`
+	Selector metav1.LabelSelector `json:"selector,omitempty"`
```

Comment:
How to I specify the namespace? Is this what the namespace on the Resource is for? What if the Resource contains resources in different namespaces?

Can I lookup a resource by name?